ADRs.md

```diff
@@ -0,0 +1,94 @@
+# Architectural Decision Records (ADRs)
+
+An Architectural Decision (AD) is a justified software design choice that addresses a functional or
+non-functional requirement that is architecturally significant. ([adr.github.io](https://adr.github.io/))
+
+## Process
+
+Architectural decisions we make in the developer meetings are recorded here. If the decision is 
+small and uncontroversial, but yet important and can be expressed in maximum 2 sentences, we will 
+list it here without any reference. If the decision require a bit more discussion and explanations
+an issue on GitHub should be created - use the template below.
+
+### How to discuss and document an Architectural Decision
+
+ - [Create a new issue](https://github.com/opentripplanner/OpenTripPlanner/issues/new/choose?template=adr.md) 
+using the [ADR.md](https://github.com/opentripplanner/OpenTripPlanner/blob/dev-2.x/ISSUE_TEMPLATE/ADR.md)
+template.
+ - Make sure to update the main description based on the feedback/discussion and decisions in the 
+   developer meeting.
+ - The final approval is done in the developer meeting, at least 3 developers representing 3 
+   different organisations should approve - and no vote against the proposal. If the developers
+   are not able to agree the PLC can decide.
+ - Add the ADR to the list below. Maximum two sentences should be used - try to keep it as short as 
+   possible. Remember to link to the discussion.
+ - References to Architectural Decision Records in reviews can be done by linking or just typing 
+   e.g. `ADR2`. Use the `[ADR1]()`
+
+
+## Records
+
+### ADR-0 Scout rule
```

Comment:
As titles of decisions, and as sub-headings, it feels to me like these should be in Title Case (in English, most words capitalized). https://en.wikipedia.org/wiki/Title_case

Reply:
:+1: for Title Case